assets/xml/objects/object_shopnuts.xml

```diff
@@ -1,26 +1,26 @@
 <Root>
     <File Name="object_shopnuts" Segment="6">
         <!-- Business Scrub Skeleton -->
-        <Skeleton Name="gBusinessScrubSkel" Type="Flex" LimbType="Standard" Offset="0x41A8"/>
+        <Skeleton Name="gBusinessScrubSkel" Type="Flex" LimbType="Standard" LimbNone="BIZ_SCRUB_LIMB_NONE" LimbMax="BIZ_SCRUB_LIMB_MAX" EnumName="BizScrubLimb" Offset="0x41A8"/>
```

Comment:
I'd prefer "business scrub" over "biz scrub" personally
and maybe it'll change to "deku salesman" eventually as it's called in code :shrug: for now consistency with the current assets names seems enough